### PR TITLE
[NL] Per Chart Messaging UI

### PR DESCRIPTION
**Pull request by chejennifer**
These are the UI changes to add plumbing for per chart messaging:
- updates BlockContainer to render an info message from the config
- update UserMessage to render a list of messages
- update styling of UserMessage to match adriana's mocks: https://www.figma.com/file/bK4jZkyB0FqfMhA1AUZVuu/Data-Commons---2024?type=design&node-id=265-7436&mode=design&t=urLGmeO8OQgk0KdL-0

Server side changes in separate PR: https://github.com/datacommonsorg/website/pull/3909

block level messages:
Screenshot 2024-01-26 at 11 45 40 AM

page level message that comes from every block having the same message:
Screenshot 2024-01-26 at 11 51 26 AM

when there was already a page level message, but every block has the same message:
Screenshot 2024-01-26 at 11 51 43 AM

### CODE REVIEW

File: static/css/shared/story_block.scss [hunk 1]
```diff
@@ -63,3 +63,16 @@ $subtopic-color: #222;
...
+.block .block-msg {
+  padding: 0 5px 25px 5px;
+
+  mark {
+    color: var(--gm-3-ref-neutral-neutral-20);
+    font-size: 16px;
+    font-style: italic;
+    font-weight: 500;
+    line-height: 24px;
+    background: var(--gm-3-ref-primary-primary-95);
```
Comment (juliawu) [inline static/css/shared/story_block.scss:76]: nit: please alphabetize

**Review by juliawu — APPROVED** [review]
Nice! Very clean.

File: static/css/shared/story_block.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/shared/story_block.scss]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**